tests/utils/test_builders.py

```diff
@@ -48,6 +48,14 @@ def test_tokenizer_builder(tokenizer_name: str, tokenizer_kwargs: dict):
         assert isinstance(tokenizer, PreTrainedTokenizerBase)
 
 
+def test_tokenizer_no_EOS(tokenizer_name: str, tokenizer_kwargs: dict):
+    with pytest.raises(ValueError, match=r".*must have an eos_token_id.*"):
+        build_tokenizer('bert-base-uncased', {})
+    
+    with pytest.raises(ValueError, match=r".*must have an eos_token_id.*"):
+        build_tokenizer('bert-base-uncased', {'eos_token_id': None})
```

Comment:
looks like for `bert-base-uncased` eos_token_id is already None by default, is there a reason to do this?